hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.test.js

```diff
@@ -1,9 +1,24 @@
 import React from "react";
 import { render } from "@testing-library/react";
-import Navbar from "./Navbar";
+import { Navbar, NavbarDrawer } from "./Navbar";
 import { withRouter } from "testing";
 
-it("renders correctly when all icons appear", () => {
-    const { asFragment } = render(withRouter(<Navbar />));
-    expect(asFragment()).toMatchSnapshot();
+describe("<Navbar />", () => {
+    it("renders correctly when all icons appear", () => {
+        const { getByText } = render(withRouter(<Navbar cartQuantity={1} />));
+        expect(getByText("Dashboard")).toBeInTheDocument();
```

Comment:
You're missing buttons here it seems (orders, teams, reports, inventory). I think you'd be fine just using a snapshot for these tests.